test/e2e/self_hosted_test.go

```diff
@@ -65,3 +65,18 @@ var _ = Describe("When testing Cluster API working on self-hosted clusters using
 		}
 	})
 })
+
+var _ = Describe("When testing Cluster API working on single-node self-hosted clusters using ClusterClass [ClusterClass]", func() {
```

Comment:
I wonder if we should drop`When testing Cluster API working on self-hosted clusters using ClusterClass [ClusterClass]` now.

I'm not sure if the 1+1 test is needed anymore and we have to keep an eye on the amount of tests we run.

@fabriziopandini WDYT?

Reply:
I'd be okay with dropping the other one. Waiting for @fabriziopandini to confirm / decline.

Reply:
Okay then let's merge as is